week2/homework/src/resetToDos.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { loadToDos, saveToDo, logScreen } = require('./Util');
+
+const resetToDos = async () => {
+  const todos = await loadToDos();
```

Comment:
no need to read any todos here, you just have to set the todos at write to be an empty array, same result and simpler way.

Reply:
Yes, absolutely you are right. Thanks.